app/uk/gov/hmrc/customs/declarations/information/controllers/VersionController.scala

```diff
@@ -73,10 +73,13 @@ class VersionController @Inject()(val shutterCheckAction: ShutterCheckAction,
       Right()
     } else {
       val appropriateResponse = if (mrn.valueTooLong) {
-        ErrorResponse(BAD_REQUEST, BadRequestCode, "MRN too long")
+        logger.info(s"MRN parameter passed was too long: $mrn")
+        ErrorResponse(BAD_REQUEST, BadRequestCode, "MRN parameter too long")
       } else if (mrn.valueTooShort) {
+        logger.info(s"MRN parameter passed was missing: $mrn")
```

Comment:
It's either passed or it's missing! Maybe `Missing MRN parameter`?

Reply:
How about:
```
MRN parameter is too long
MRN parameter is missing
MRN parameter is invalid
```